yaml/core/of-auth-dep.yml

```diff
@@ -34,6 +34,10 @@ spec:
             value: ""
           - name: client_id
             value: ""
+          - name: oauth_provider_base_url
+            value: "https://gitlab.o6s.io"
```

Comment:
Is this right? ^

Reply:
That is just example value, but I can remove it

Reply:
Changed + comment added